examples/docs_beta_snippets/docs_beta_snippets/guides/data-assets/quality-testing/freshness-checks/external-asset-freshness-check.py

```diff
@@ -0,0 +1,55 @@
+from datetime import timedelta
+
+import dagster_snowflake as dg_snowflake
+
+import dagster as dg
+
+
+@dg.observable_source_asset(specs=[dg.AssetSpec("hourly_sales")])
+def hourly_sales(snowflake: dg_snowflake.SnowflakeResource):
```

Comment:
+1

Reply:
I actually expect both names to be the same, since the asset spec is just defining an asset key...

But that actually reminds me that defining an asset spec here is redundant, so I'm removing that